src/components/AddHouse.jsx

```diff
@@ -0,0 +1,219 @@
+          <form onSubmit={handleSubmit} className="w-100 mt-5 d-flex add-form">
+            <div className="mb-3">
+              <label htmlFor="houseName" className="form-label">
+                <input
+                  type="text"
+                  name="name"
+                  value={houseDatas.name}
+                  onChange={handleChange}
+                  id="houseName"
+                  className="form-control"
+                  placeholder="Enter House Name"
+                />
+              </label>
+            </div>
+            <div className="mb-3">
+              <label htmlFor="houseArea" className="form-label">
+                <input
+                  type="number"
+                  name="area"
+                  value={houseDatas.area}
+                  onChange={handleChange}
+                  className="form-control"
+                  id="houseArea"
+                  placeholder="Enter House area"
+                />
+              </label>
+            </div>
+
+            <div className="mb-3">
+              <label htmlFor="housePrice" className="form-label">
+                <input
+                  type="number"
+                  name="price"
+                  value={houseDatas.price}
+                  onChange={handleChange}
+                  className="form-control"
+                  id="housePrice"
+                  placeholder="Enter House Price"
+                />
+              </label>
+            </div>
+            <div className="mb-3">
+              <label htmlFor="houseDescription" className="form-label">
+                <input
+                  type="text"
+                  name="description"
+                  value={houseDatas.description}
+                  onChange={handleChange}
+                  className="form-control"
+                  id="houseDescription"
+                  placeholder="Enter House Description"
+                />
+              </label>
+            </div>
+            <div className="mb-3">
+              <label htmlFor="houseRooms" className="form-label">
+                <input
+                  type="number"
+                  name="number_of_rooms"
+                  value={houseDatas.number_of_rooms}
+                  onChange={handleChange}
+                  className="form-control"
+                  id="houseRooms"
+                  placeholder="Enter Number of Rooms"
+                />
+              </label>
+            </div>
+
+            <div className="mb-3">
+              <label htmlFor="houseLocation" className="form-label">
+                <input
+                  type="text"
+                  name="location"
+                  value={houseDatas.location}
+                  onChange={handleChange}
+                  className="form-control"
+                  id="houseLocation"
+                  placeholder="Enter the location of the house"
+                />
+              </label>
+            </div>
+
```

Comment:
These input/label combinations are repetitive and can be abstracted futher with Subcomponents, array or similar and some sort of mapping function

Reply:
@mcrd25 Thank you very much for your constructive comment